src/features/people/PersonPage/index.js

```diff
@@ -31,12 +25,6 @@ export default () => {
     }
   }, [dispatch, id, language]);
 
-  useEffect(() => {
-    if (query) {
-      history.push(`/people?${searchParams.toString()}`);
-    }
-  }, [query, history, searchParams]);
-
```

Comment:
w całej aplikacji się namęczyliśmy z tymi useEffectami a rzeczywiście można było to do sagi wsadzić tak jak to zrobiłeś ;)